code-challenges.js

```diff
@@ -47,3 +182,19 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+// Function Signature
+// input: larger numbers and smaller numbers
+// output: smaller numbers being subtracted from larger numbers
+
+// Example
+// input: 100-58 // output --> 42
+// input: 27-24 // output --> 3
+
+// Process
+// create a console.log that will subtract the smaller number from the larger number
+// the first console log will take number2 (100) and subtract number1 (58) from it to get the outcome of 42
+// the second console log will take number3 (27) and subtract number4 (24) from it to get the outcome of 3
+
+console.log(number2-number1)
+console.log(number3-number4)
```

Comment:
Okay this one calls for a function  that will use conditionals to determine which larger number needs to have a smaller number subtracted from it and all you are doing is console logging some math. Your pseudocode is good but without that one little detail you miss the opportunity.  I recommend you give yourself the win and come back to this one at some point and write out the function!